### PR TITLE
View Submissions UI

### PR DESCRIPTION
**Pull request by maziasy**
- [x] Hover over the images and click on one to see it in full resolution, and (potentially) declare it as the winner
- [ ] Optional: Add button next to contest title to view contest brief (in a pop-up)!

ViewSubmissions
ViewFull

### CODE REVIEW

**Review by john-rigas — APPROVED** [review]
Everything looks good to me!

File: client/src/pages/AccountButton.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    margin: theme.spacing(1),
+  	"margin-right" : "2%",
+  },
+  input: {
+    display: 'none',
+  },
+  title: {
+  	"font-family" : theme.fontFamily,
+  	"font-size" : "10px",
+  	padding : "5px"
```
Comment (ronald-so) [inline client/src/pages/AccountButton.js]: indents and camel case for consistency!

File: client/src/pages/BackToContestsListsButton.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    margin: theme.spacing(1),
+  	"margin-right" : "2%",
+  },
+  input: {
+    display: 'none',
+  },
+  title: {
+  	"font-family" : theme.fontFamily,
+  	"font-size" : "10px",
+  	padding : "5px"
```
Comment (ronald-so) [inline client/src/pages/BackToContestsListsButton.js]: same as above

File: client/src/pages/ImagesGrid.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+const useStyles = makeStyles(theme => ({
+	root: {
+	  display: 'flex',
+	  flexWrap: 'wrap',
+	  overflow: 'hidden',
+    backgroundColor: theme.palette.background.paper,
+    "padding-left": "0.75%",
+    "padding-right": "0.5%",
+	},
```
Comment (rwhs) [inline client/src/pages/ImagesGrid.js]: same as above!

File: client/src/pages/ImagesGrid.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	gridList: {
+	  width: '100%',
+    height: '100%',
+    "padding-top": "1%",
+  },
+  tilebar: {
+    "height":"12.5%",
+    "margin-right": "1.5%",
+    "margin-bottom": "1.5%"
```
Comment (ronald-so) [inline client/src/pages/ImagesGrid.js]: same as above!

**Review by ronald-so — APPROVED** [review]
just some small formatting things, everything else looks great though!

File: client/src/pages/AccountButton.js (hunk 1 above)
Comment (maziasy) [reply client/src/pages/AccountButton.js]: Thanks Ronald! This file has actually been deleted in the latest commit (I think it still shows up when you view all commits), but I'll make the formatting changes for the other files :)

**Pull request merged**